pennylane_qiskit/qiskit_device2.py

```diff
@@ -207,6 +209,18 @@ class QiskitDevice2(Device):
             with those in kwargs, the settings in `options` will take precedence over kwargs. Keyword
             arguments accepted by both the transpiler and at runtime (e.g. ``optimization_level``)
             will be passed to the transpiler rather than to the Primitive.
+
+    .. details::
+        :title: Tracking
+
+        ``QiskitDevice2`` tracks:
+
+        * ``executions``: the number of unique circuits that would be required on quantum hardware
+        * ``shots``: the number of shots
+        * ``resources``: the :class:`~.resource.Resources` for the executed circuit.
+        * ``simulations``: the number of simulations performed. One simulation can cover multiple QPU executions, such as for non-commuting measurements and batched parameters.
+        * ``batches``: The number of times :meth:`~.execute` is called.
+        * ``results``: The results of each call of :meth:`~.execute`
```

Comment:
Do we want the tracker on this device to distinguish between `simulations` and `executions` like the simulator tracker does? And have we thought about how much something like `executions` means when with `Estimator` we send things along to Qiskit to let them deal with non-commuting measurements, etc?

Reply:
>Do we want the tracker on this device to distinguish between simulations and executions like the simulator tracker does? 

Hmm I don't think it distinguishes between simulations and executions right now. I'm actually not sure if Qiskit distinguishes simulations/executions between running on Aer vs. on real hardware. In any case I don't think it's important/useful to distinguish between simulations and executions. It might be a bit confusing though to have `simulations` exist though.

>And have we thought about how much something like executions means when with Estimator we send things along to Qiskit to let them deal with non-commuting measurements, etc?

Counting `Executions` are different for the QiskitDevice compared to default.qubit when given non-commuting measurements -- so right now each it's been split, it's counted as a separate execution.